tests/conftest.py

```diff
@@ -212,6 +219,49 @@ def s3_rig(request, monkeypatch, assets_dir):
         bucket.objects.filter(Prefix=test_dir).delete()
 
 
+@fixture()
+def custom_s3_rig(request, monkeypatch, assets_dir):
```

Comment:
Let's add a docstring that this is used for testing things like MinIO/ceph?

Reply:
done here fc113dc